saltax/plugins/f_raw_records.py

```diff
@@ -95,11 +101,10 @@ def compute(self, raw_records, start, end):
             chunk_data = chunk_data.to_records(index=False)
             data = np.zeros(len(chunk_data), dtype=self.dtype)
             strax.copy_to_buffer(chunk_data, data, "_bring_data_into_correct_format")
-
             self.source_done = source_done
 
             # Stick rigorously with raw_records time range
-            return self.chunk(start=start, end=end, data=data, data_type="geant4_interactions")
+            return self.chunk(start=start, end=end, data=data)#, data_type="geant4_interactions")
```

Comment:
It would be nice if you can direct me to where your records/raw_records_simu is

Reply:
Honestly I am not sure if it is necessary and removing it will break things. I am just a bit nervous since this is chunk related, and chunk size does matter hugely for memory consumption. I see that your memory requested is already 55GB. I would say 70GB is the maximum we can take. Maybe give it a last try with 70GB? 
Also I would want to see the raw_records_simu and records files. Is it somewhere on midway?

Reply:
No sorry I keep them in my scratch. records chunk size is ~1.2GB each with 72 chunks in total and, raw_records_simu chunk size is ~300MB and also 72 chunks in total. Is that helpful?

Yes I would have already gone for more memory but broadwl won't give me more than 58GB and xenon1t partition can't see project and midway3/scratch.

Reply:
_No sorry I keep them in my scratch. records chunk size is ~1.2GB each with 72 chunks in total and, raw_records_simu chunk size is ~300MB and also 72 chunks in total. Is that helpful?_

This is helpful. Maybe try `lgrandi` on midway3. Note that you will have to reinstall things... which gonna be a painful 5 minutes

Reply:
From the info you gave I don't think this is where things break. Maybe it is just the AmBe super heavy nature itself + some memory leakage. We never figured out why peak building is so expensive in memory.

Reply:
By the way, it worked with 65GB on lgrandi. Also, I wasn't aware that the one run I tested with is a 20min AmBe run from topCW5d9m position, while the topCW7d8m ones are only 10 minutes. So probably this is really just super heavy AmBe 😐